README.md

```diff
@@ -1,5 +1,33 @@
 **⚠️ UNDER CONSTRUCTION ⚠️**
 
+Make an order through API CowSwap
+
+Dependencies
+python3 version 3.6 or greater
+eip712
+hexbytes
+web3.auto
+
+Installation
+pip install eip712
+pip install hexbytes
+pip install web3
```

Comment:
Once the dependencies are in `requirements.txt` file we can run `pip install -r requirements.txt`

Reply:
added it